src/unitxt/metrics.py

```diff
@@ -4731,6 +4731,50 @@ def compute(self, references: List[Any], prediction: Any, task_data: Dict) -> di
         return {self.main_score: prediction}
 
 
+class F1Strings(InstanceMetric):
+    main_score = "f1_strings"
+    reduction_map = {"mean": ["f1_strings"]}
+    prediction_type = str
+    single_reference_per_prediction = True
+    _requirements_list = {
+        "spacy": "Please pip install spacy",
+        "en_core_web_sm": "Please run: python -m spacy download en_core_web_sm",
+    }
+
+    def prepare(self):
+        super().prepare()
+        import spacy
+
+        self.nlp = spacy.load("en_core_web_sm")
```

Comment:
```suggestion
        try:
             self.nlp = spacy.load("en_core_web_sm")
        except OSError:
            from spacy.cli import download
            download('en')
            self.nlp = spacy.load("en_core_web_sm")
```
This means we can remove "en_core_web_sm" from the requierements list as it will be downloaded automatically when missing